code/modules/deathmatch/deathmatch_loadouts.dm

```diff
@@ -0,0 +1,374 @@
+/datum/outfit/deathmatch_loadout //remember that fun > balance
+	name = ""
+	shoes = /obj/item/clothing/shoes/sneakers/black // im not doing this on all of them
+	/// Name shown in the UI
+	var/display_name = ""
+	/// Description shown in the UI
+	var/desc = ":KILL:"
+	/// If defined, using this outfit sets the targets species to it
+	var/datum/species/species_override
+	/// This outfit will grant these spells if applied
+	var/list/granted_spells = list()
+
+/datum/outfit/deathmatch_loadout/pre_equip(mob/living/carbon/human/user, visualsOnly = FALSE)
+	. = ..()
+	if(istype(user, /mob/living/carbon/human/dummy))
+		return
+
+	if(!isnull(species_override))
+		user.set_species(species_override)
+	if(islist(granted_spells) && granted_spells.len)
+		for(var/datum/action/act as anything in granted_spells)
+			GRANT_ACTION(act)
```

Comment:
```suggestion
	for(var/datum/action/act as anything in granted_spells)
		GRANT_ACTION(act)
```
`for(var/thing in null) and `for(var/thing in list())` are both valid, they're just NO-OPs.